doc/exclude_patterns.inc

```diff
@@ -1,7 +1,5 @@
 # The following patterns are to be excluded from the documentation build
 tests
-modules
-sensors
 build
 
 **README*
```

Comment:
I think you need to include the top level readme for pdf but exclude all other readmes